mbuild/formats/lammpsdata.py

```diff
@@ -1,6 +1,9 @@
 from __future__ import division
 
 from collections import OrderedDict
+from parmed.parameters import ParameterSet
```

Comment:
nit: Can you separate the standard library and third-party imports and arrange in alphabetical order?